app/templates/plans.hbs

```diff
@@ -100,7 +100,7 @@
         </li>
         <li>
           <h4 class="h3--teal">What hardware are my tests running on?</h4>
-          <p class="faq-text">We use dedicated hardware to ensure full speed for your tests. You have 3GB of memory and up to 2 cores available. Your tests are running directly off SSDs with the most common databases optimized to run off RAM disks.</p>
+          <p class="faq-text">You have 3GB of memory and up to 2 cores available. Your tests are running directly off SSDs with the most common databases optimized to run off RAM disks.</p>
```

Comment:
Just got from feedback from @joshk that we can just remove the entire `<li>` concerning hardware.